stores/browser_store.tsx

```diff
@@ -64,11 +67,11 @@ class BrowserStoreClass {
         }
     }
 
-    isSignallingLogin(loginId) {
+    isSignallingLogin(loginId: string) {
         return loginId === sessionStorage.getItem(StoragePrefixes.LOGIN);
     }
 
-    clear(options) {
+    clear(options?: any) { // TODO add more specific types
```

Comment:
I think this type should work here
```suggestion
    clear(options?: {exclude?: string[]}) {
```

Reply:
Getting this error when adding this type annotation:

```
Argument of type '{ exclude?: string[] | undefined; } | undefined' is not assignable to parameter of type '{ exclude: never[]; } | undefined'.
  Type '{ exclude?: string[] | undefined; }' is not assignable to type '{ exclude: never[]; }'.
    Types of property 'exclude' are incompatible.
      Type 'string[] | undefined' is not assignable to type 'never[]'.
        Type 'undefined' is not assignable to type 'never[]'.ts(2345)
```

Note that `actions/storage.js` is a javascript file, and the signature of `Actions.clear` is
```js
export function clear(options = {exclude: []}) {
```

Reply:
That's fine then. We can always fix it once more of that is migrated to TS.